src/components/register/RegisterForm.vue

```diff
@@ -0,0 +1,261 @@
+                aria-required="true"
+                class="field__input"
+                @focus="resetError()"
+                @blur="check('email')"
+                v-model="email.value"
+            />
+            <p
+                id="email-constraints"
+                class="field__error"
+                v-if="hasError('email')"
+            >
+                Email is required
+            </p>
+        </div>
+        <div class="field">
+            <label class="field__label" for="password">Password</label>
+            <b-form-input
+                id="password"
+                type="password"
+                autocomplete="current-password"
+                aria-describedby="password-constraints"
+                required
+                aria-required="true"
+                class="field__input"
+                @focus="resetError()"
+                @blur="check('password')"
+                v-model="password.value"
+            />
+            <p
+                id="password-constraints"
+                class="field__error"
+                v-if="hasError('password')"
+            >
+                Password is required
+            </p>
+        </div>
+        <div class="field">
+            <label class="field__label" for="password2">Confirm password</label>
+            <b-form-input
+                id="password2"
+                type="password2"
+                autocomplete="current-password"
+                aria-describedby="password-constraints"
+                required
+                aria-required="true"
+                class="field__input"
+                @focus="resetError()"
+                @blur="check('confirm_password')"
+                v-model="confirmpassword.value"
+            />
+            <p
+                id="password-constraints"
+                class="field__error"
+                v-if="hasError('password')"
+            >
+                Please, confirm your password
+            </p>
+        </div>
+        <div class="field">
+            <input type="checkbox" id="checkbox" v-model="checked" />
+            <label>
+                I have read and accept the
+                <a
+                    target="_blank"
+                    class="field__action"
+                    href="https://www.plasticorigins.eu/CGU"
+                    >Terms of use and Privacy Policy
+                </a>
+            </label>
+        </div>
+        <button
+            type="submit"
+            class="btn btn-primary mt-6"
+            v-bind:disabled="!isValid || status === 'loading'"
+        >
+            {{ status === "loading" ? "Loading" : "Register" }}
+        </button>
+        <b-card class="bg-danger mt-3" v-if="status === 'error'">
+            <b-card-text>An error occurred. Please try again.</b-card-text>
+        </b-card>
+        <b-card class="bg-success mt-3" v-if="status === 'fulfilled'">
+            <b-card-text>Register in. Please wait...</b-card-text>
```

Comment:
Pas sûr que register in se dise :) 
```suggestion
            <b-card-text>Logged in. Please wait...</b-card-text>
```